defining-your-schema.mdx

```diff
@@ -141,6 +141,20 @@ type Comment {
 }
 ```
 
+```
+type Direction enum {
+    Up
+    Down
+    Left
+    Right
+}
+
+type Car {
+    brand: string
+    direction: Direction?
+}
+```
```

Comment:
I think this should:
- be merged to the above code block; or
- replace the above code block; or
- extend the above code block

Reply:
fixed